test/test.js

```diff
@@ -48,6 +48,19 @@ $.mockjax( {
 	}
 } );
 
+$.mockjax( {
```

Comment:
maybe we should even move the `$.mockjax` call into the test which is using it..?